CRM/Activity/Import/Parser/Activity.php

```diff
@@ -151,23 +151,14 @@ public function summary(&$values) {
     $this->setActiveFieldValues($values, $erroneousField);
     $index = -1;
 
-    if ($this->_activityTypeIndex > -1 && $this->_activityLabelIndex > -1) {
-      array_unshift($values, ts('Please select either Activity Type ID OR Activity Type Label.'));
```

Comment:
I read this as NOT being a duplicate of what happens in mapfield formRule. And when I run I get two different things before and after the patch. What this line is saying is **you can't select BOTH id and label** for activity type in the same mapping. Whereas in mapfield it's checking that you have to pick at least one. The error message here is a little ambiguously worded.

Reply:
Ah OK - I suppose that is a valid error - I hadn't picked that up from the code - but we should probably only error if they conflict